Georeferencing-Tin-12d-1/readme.md

```diff
@@ -1,14 +1,10 @@
 
-# UT-Georeferencing-Tin-12d-1 
-
-Tin in map coordinates
-
 ### Intent
 
 IFC 4x3 RC2
 This scenario has a tin (only 1.5K triangles) with just one colour for all the triangles. 
 The coordinates are in Easting, Northing and AHD for the map projection MGA94 Zone 56.
-The Horizontal and Vertical datums are specified in IfcProjectedCRS.
+The Horizontal and Vertical datums are specified in 'IfcProjectedCRS'.
```

Comment:
That's the wrong char denoting "code snippets" in Markdown.

You're using a single quotation mark ' - it should rather be the accent, described as "from top left to right down", e.g. as used for "è". 

Observe the char as used for the file names in the readmes' tables at the bottom.